BigBaseV2/src/util/vehicle.hpp

```diff
@@ -6,6 +6,7 @@
 #include "pointers.hpp"
 #include "script.hpp"
 #include "teleport.hpp"
+#include <script_global.hpp>
```

Comment:
Any reason you used < > instead of " " for the include?

Reply:
Actually no, I just copied my shit over didn't see but if it triggers you, change it